ground/src/main/java/com/google/android/ground/persistence/remote/firebase/protobuf/ModelToProtoExt.kt

```diff
@@ -139,6 +139,7 @@ private fun toTaskData(id: String, newTaskData: TaskData) = taskData {
     is MultipleChoiceTaskData -> multipleChoiceResponses = multipleChoiceResponses {
         newTaskData.getSelectedOptionsIdsExceptOther().forEach { selectedOptionIds.add(it) }
         if (newTaskData.hasOtherText()) {
```

Comment:
Can we rename `hasOtherText()` to `isOtherSelected()` for correctness?

Reply:
not a part of this PR, but let me update at all places

Reply:
done